bluesky_kafka/produce.py

```diff
@@ -135,16 +136,16 @@ def __str__(self):
             safe_config["sasl.password"] = "****"
         return (
             f"{type(self)}("
-            f"topic='{self.topic}',"
-            f"key='{self._key}',"
-            f"bootstrap_servers='{self._bootstrap_servers}'"
+            f"topic='{self.topic}', "
+            f"key='{self._key}', "
+            f"bootstrap_servers='{self._bootstrap_servers}', "
             f"producer_config='{safe_config}'"
```

Comment:
Do you want the servers' list and the config to be in single quotes? They will be treated as strings rather than as list/dict.

Reply:
Check the demo in my earlier comment/review: https://github.com/bluesky/bluesky-kafka/pull/51#discussion_r1108777766.

Reply:
cleaned that up for both of the new classes